test/paypal.js

```diff
@@ -6,7 +6,8 @@ import * as paypalCheckout from '../src/interface/button'; // eslint-disable-lin
 import * as paypalWallet from '../src/interface/wallet'; // eslint-disable-line import/no-namespace
 import * as paypalMarks from '../src/interface/marks'; // eslint-disable-line import/no-namespace
 
-insertMockSDKScript();
+// the enable-funding=venmo flag is temporarily needed for the venmo experiment
+insertMockSDKScript({ query: { 'client-id': 'test', 'enable-funding': 'venmo' } });
```

Comment:
Let me know if there is a better way to mock this. I tried mocking the experiment state with `window.localStorage.setItem('enable_venmo', true)` but it would not work when being added to the multiple.js test file. Mocking the enable-funding value here seemed like the easiest way to get the integration tests passing.

Reply:
👍